src/swsssdk/scripts/sonic-db-cli

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+import sys
+import swsssdk
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.NullHandler())
```

Comment:
Do we really need logging? Is it a performance issue?

Reply:
I don't think it is necessary. There is no logging in original redis-cli shell script. I think we can remove it. Definitely it will add overhead to run this python script.

Reply:
removed